src/pages/patterns/disclosures-pattern/index.mdx

```diff
@@ -0,0 +1,779 @@
+---
+title: Disclosures
+description:
+  Disclosures are small moments that open up on a page and reveal additional
+  information related to the source it is triggered from.
+---
+
+<PageDescription>
+
+Disclosures are small moments that open up on a page and reveal additional
+information related to the source it is triggered from.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Designing with disclosures</AnchorLink>
+<AnchorLink>Profile menu</AnchorLink>
+<AnchorLink>Context menu</AnchorLink>
+<AnchorLink>Settings and filter menus</AnchorLink>
+<AnchorLink>Combo button</AnchorLink>
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>References</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Disclosures support a wide range of different use cases in product interfaces
+and are commonly used to reveal more information or details about an element or
+content on a page. Disclosures can contain interactive elements while preserving
+accessible functionality.
```

Comment:
Maybe: "Unlike tooltips, the content expanded by a Disclosure may contain interactive elements."

I think the important thing to clarify is that if the Disclosure is two parts: trigger + content, then the content is what may have interactive elements without removing semantics like a tooltip would.